src/olympia/activity/models.py

```diff
@@ -39,6 +39,17 @@
 # Number of times a token can be used.
 MAX_TOKEN_USE_COUNT = 100
 
+GENERIC_USER_NAME = gettext('Add-ons Review Team')
+
+
+class GenericMozillaUser(UserProfile):
+    class Meta:
+        proxy = True
+
+    @property
+    def name(self):
+        return GENERIC_USER_NAME
+
```

Comment:
not required for this patch, but if we start to use this class more widely we should override the save method (and possibly others) to raise if it's inadvertently written to the database or used in unexpected ways.

Reply:
Good idea. Should I file an issue about that, or shall we just wait to see if we end up using it for something else and address it at that time?

Reply:
I'd just wait